wordpress/wp-content/plugins/memberful-wp/src/user/map.php

```diff
@@ -146,7 +146,11 @@ public function ensure_present() {
     $user_id = $this->user_exists ? $this->update_user() : $this->create_user();
 
     if ( is_wp_error( $user_id ) ) {
-      $user_id->add_data( array_merge( (array) $user_id->get_error_data(), compact('user_data') ) );
+      if (isset($user_data)) {
```

Comment:
I wonder what scope `isset()` uses. `$user_data` is not set in this function's scope so I presume that `isset($user_data)` will never return `true`. I'll probably play with it later.

Reply:
`$user_data` used to be set in the function's scope, but https://github.com/memberful/memberful-wp/commit/bad84b0fd0331972a16f16c4bb3ab1ba14dccaba#diff-b9bd8651174701b4c1f2fdd53734864ae83e9cafd63235fa81124b1a49f60ffbL153 removed it.

Therefore, I presume we can get rid of the `compact()` call.